react/Modal/index.jsx

```diff
@@ -86,8 +83,8 @@ class ModalContent extends Component {
   render() {
     const { className } = this.props
 
-    const { displayGhostHeader, animatedHeader, childrenToRender } = this.state
```

Comment:
By using `this` if we have an update in the `animatedModalHeader` component it won't update the content here :/. It's why I used the state finally.